client/packages/host/src/components/Login/Login.tsx

```diff
@@ -98,7 +102,7 @@ export const Login = () => {
         error && (
           <ErrorWithDetails
             error={error.message || t('error.login')}
-            details={error.detail || ''}
```

Comment:
b2a2a6262. Now it would show `Account is blocked until the lockout period has expired MM:SS` with real time updating of timeoutRemaining in the login page.